src/olympia/addons/tests/test_models.py

```diff
@@ -338,7 +334,7 @@ def test_filter_for_many_to_many(self):
         assert collection.addons.count() == 0
 
         # Only deleted.
-        self.addon.update(is_listed=True)
+        self.make_addon_unlisted(self.addon, listed=True)
```

Comment:
The fact that `make_addon_unlisted()` can take a `listed` argument is super weird. Can we just have a `make_addon_listed()` method instead and use that ?